pkg/oc/cli/describe/projectstatus.go

```diff
@@ -164,6 +164,37 @@ func (d *ProjectStatusDescriber) MakeGraph(namespace string) (osgraph.Graph, set
 	return g, forbiddenResources, nil
 }
 
+func selectorEquals(A, B map[string]string) bool {
```

Comment:
Have you looked at `k8s.io/kubernetes/staging/src/k8s.io/apimachinery/pkg/labels` to compare selectors and labels? I think you could reuse some of the logic from there.